### DISCUSSION TITLE
Raspberry Pi 4 gesucht

### DISCUSSION THREAD

**Discussion by camueller** [camueller/SmartApplianceEnabler#311] [Allgemein]
Zur Weiterentwicklung des SAE nutze ich einen Raspberry Pi 4, aber im produktiven Einsatz habe ich zwei Raspberry Pi 2 Model B. Einen davon würde ich gern auf Raspberry Pi 4 umstellen, damit ich sicher sein kann, dass auch im Alltag der Zugriff auf GPIO funktioniert. Die aktuellen Preise halten mich allerdings davon ab. Wenn jemand von Euch einen Raspberry Pi 4 günstig abgeben möchte oder diesen gegen einen Raspberry Pi 2 Model B tauschen möchte - bitte melden.

**Comment by bobmorane77** [comment]
Als Idee
Raspberry PC Pi 400
derzeit 50€ zu sparen, bessere Kühlung hat er gleich mit dabei

**Reply by camueller** [reply]
Wäre eine Option, aber vielleicht finde ich auch noch einen normalen Pi 4 ohne Dinge zu kaufen, die ich eigentlich nicht brauche.

**Comment by camueller** [comment]
Vielen Dank an Maik Zober, der seinen nicht mehr benötigten RaspberryPi 4 gespendet hat :-)

Somit habe ich einen meiner produtkiven Raspis durch diesen ersetzt und kann dadurch sicher sein, dass auf Raspberry Pi 2 + 4 das GPIO-Handling in der Praxis korrekt funktioniert.

**Comment by AndreasKzk** [comment]
Ist das noch aktuell? Ich hätte noch ein Gerät.

**Comment by camueller** [comment]
Vielen Dank - ich benötige aktuelle keinen weiteren Pi4. Wenn Du ihn bei Ebay verkaufst, wirst Du ein reicher Mann :-)
